### DISCUSSION TITLE
Feature Request: Separate photos  and video

### DISCUSSION THREAD

**Discussion by AnilRh** [ivandokov/phockup#127] [Ideas]
It would be great to be able to organize by type of file too. For example:
/video/2020/01/bob.mov
/photo/2020/01/bob.jpg

The video collection often needs more pruning than the photo collection, because of its size. Yes, it should be pretty easy to split the files by extension before running phockup - but think it would still be a nice addition.

Great tool btw - thank you!

**Comment by ivandokov** [comment]
That's a great idea. In fact we can process audio files too. PR is more than welcome!
